src/stories/components/Axis/Axis.test.tsx

```diff
@@ -191,6 +191,15 @@ describe('Axis', () => {
 			expect(screen.getByText('$2M')).toBeInTheDocument();
 			expect(screen.getByText('$500K')).toBeInTheDocument();
 		});
+		test('shortCurrency with small range', async () => {
+			render(<NumberFormat {...NumberFormat.args} numberFormat="shortCurrency" range={[0, 0.1]} />);
+			const chart = await findChart();
+			expect(chart).toBeInTheDocument();
+
+			expect(screen.getByText('$0.1')).toBeInTheDocument();
+			expect(screen.getByText('$0.05')).toBeInTheDocument();
+			expect(screen.getByText('$0')).toBeInTheDocument();
```

Comment:
Would having the `$0.05` check cause flakiness if the `<NumberFormat/>` component were to change default behavior? (e.g. not show a middle value, or show 3rds, etc)

It might be better to just check for the values you can reliably know will be there (`0` and `0.1`)